rust/template/differential_datalog/src/program/worker.rs

```diff
@@ -609,54 +610,79 @@ fn render_relation<'a>(
     >,
     delayed_vars: &DelayedVarMap<Child<'a, Worker<Allocator>, TS>>,
 ) {
-    // Relation may already be in the map if it was created by an `Apply` node
-    let mut collection = collections.remove(&relation.id).unwrap_or_else(|| {
-        let (session, collection) = scope.new_collection::<DDValue, Weight>();
-        sessions.insert(relation.id, session);
-
-        collection
-    });
+    scope.clone().region_named(relation.name(), |region| {
```

Comment:
So we're using regions to make relation/rule metadata visible to external tools (e.g., debuggers). I didn't realize this would be the case, but I guess it makes sense.  For operators, I was under the impression that they all now have `_named` versions, so we could use those instead, but it looks like there's only a small number of `_named` operators (`threshold_named`, `reduce_named`, and a few more).

Do you know if regions have any runtime cost?